examples/jupyter/README.md

```diff
@@ -0,0 +1,18 @@
+# Jupyter 
+It is rather simple to deploy Jupyter on MANTL. There are some official Docker images that you can use for this purpose (https://github.com/jupyter/docker-stacks). For this example we use the jupyter/minimal-notebook.
+
+
+We wrapped the Marathon submit REST call in a small script: `marathon_submit.sh`. You can use it to deploy Jupyter on your MANTL cluster running the following commands.
+
+```bash
+source bin/set_env.sh #you will asked to enter your control node hostname (without https://), and admin password 
```

Comment:
This causes sensitive information to be exported to the environment, it might be good to warn the user.  Or, you could just combine these two steps without exporting anything.